capstone-frontend/src/components/BookSearchTile.jsx

```diff
@@ -40,4 +34,31 @@ const BookSearchTile = (props) => {
   );
 }
 
+const BookListDropdownActive = (props) => {
```

Comment:
Would it be possible to make one BookListDropdown component that renders conditionally based on the length of userBookLists? We could do the conditional rendering inside the BookListDropdown instead of outside the way you do it here. Not necessary but I think it would be cleaner.

Reply:
Yup! This is a good idea. It also makes it less confusing to define two different functions with different names. Changed and confirmed it still works as expected!